### PR TITLE
Add GraphQL documentation

### PR DESCRIPTION
**Pull request by toolmantim**
This adds documentation for our GraphQL API, which I wanted to link to from an updated Scheduled Builds documentation page:

unknown-3-3-3

### CODE REVIEW

**Comment by toolmantim** [comment]
I've just updated this with a better intro, mention of Relay, and pointers to the frontend repository:

unknown-3-4-4-4-4

File: pages/graphql_api.md.erb [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# GraphQL API Overview
+
+The Buildkite GraphQL API provides an alternative to the [REST API](/docs/rest-api). It allows for more efficient retrieval of data by allowing you to fetch multiple, nested resources in a single request. The Buildkite GraphQL API is currently in beta.
```
Comment (harrietgrace) [inline pages/graphql_api.md.erb]: Double 'allow', could change the first one to 'enables'

File: pages/graphql_api.md.erb [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+<%= toc %>
+
+## Getting started
+
+The easiest way to get started with the GraphQL API is to generate a [Personal API Access Token](https://buildkite.com/user/api-access-tokens) with GraphQL scope, and then use the [GraphQL Explorer](https://graphql.buildkite.com/explorer) with its built-in documentation:
```
Comment (harrietgrace) [inline pages/graphql_api.md.erb]: 'Quickest' rather than 'easiest'?

File: pages/graphql_api.md.erb [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+<a href="https://graphql.buildkite.com/explorer"><%= image "explorer.png", width: 1065, height: 489, alt: "Screenshot of the Buildkite GraphQL Explorer" %></a>
+
+See our [Getting Started Tutorial](https://building.buildkite.com/tutorial-getting-started-with-graphql-queries-and-mutations-11211dfe5d64) for a step-by-step guide for using GraphQL queries and mutations.
```
Comment (harrietgrace) [inline pages/graphql_api.md.erb]: *to using GraphQL

**Comment by toolmantim** [comment]
Some more updates:

unknown-4-4-34-4-4

**Comment by toolmantim** [comment]
Thanks for the catches @harrietgrace! Good to go?

**Pull request merged**